requirements.txt

```diff
@@ -1,6 +1,7 @@
 django>=2.1.5,<2.2
 pytest==5.1.2
 pytest-django==3.5.1
+python-slugify==4.0.0
```

Comment:
Remember to get rid of this dependency if you decide to go with `django.utils.text.slugify`.